src/model/spec/userProfile.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { expect } from 'chai';
+import * as deepFreeze from 'deep-freeze';
+import { UserProfile } from '../userProfile'
+import {} from 'mocha'
+import {} from 'core-js'
+
+describe('userProfile', () => {
+  let userProfile = null;
+
+  beforeEach(() => {
+    userProfile = new UserProfile();
+  })
```

Comment:
Yes, I have used this on React Components, but I was not aware that it had advantages on normal objects, e.g. in this case is just an entity like class, is it worth to build it using mount?